CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 0.15.0
+
+### Features
+
+- Implemented blockList users API via `useBlockList` hook.
```

Comment:
End user does not work directly with `useBlockList `, so this can be replaced to 

```
- Implemented Block users API 
```